Tests/com/drew/imaging/jpeg/JpegMetadataReaderTest.java

```diff
@@ -57,6 +59,24 @@ public void testExtractXmpMetadata() throws Exception
         assertNotNull(directory);
     }
 
+    @Test
+    public void testFStopFormatting() throws Exception
+    {
+        Locale previousLocale = Locale.getDefault();
+
+        // Setting this locale would cause an f-number with a comma:
+        Locale.setDefault(Locale.forLanguageTag("nl-NL"));
+
+        try {
+            Metadata metadata = JpegMetadataReader.readMetadata(new File("Tests/Data/withExif.jpg"));
+            final ExifSubIFDDirectory subIFDDirectory = metadata.getFirstDirectoryOfType(ExifSubIFDDirectory.class);
+            assertEquals("f/4.0", subIFDDirectory.getDescription(ExifDirectoryBase.TAG_FNUMBER));
```

Comment:
No, because this change makes the code independent of locale. It now always returns `f/4.0`, regardless of locale. 

Would you expect these to be formatted differently for different locales?